resources/tentative/todomvc/src/components/todo-app/todo-app.template.js

```diff
@@ -0,0 +1,20 @@
+const template = document.createElement("template");
+const globals = "src/styles/global.css";
+const styles = "src/styles/app.css";
+const mainStyles = "src/styles/main.css";
+
+template.id = "todo-app-template";
+template.innerHTML = `
+    <link rel="stylesheet" href="${globals}" />
+    <link rel="stylesheet" href="${styles}" />
+    <link rel="stylesheet" href="${mainStyles}" />
+    <section class="app">
+        <slot name="topbar"></slot>
```

Comment:
I think more idiomatic thing to do is to have <todo-topbar></todo-topbar> instead and rely on custom elements to instantiate the component.

Reply:
Hopefully addressed in https://github.com/WebKit/Speedometer/pull/249#discussion_r1240598703 below